core/src/tests/test_edit.cc

```diff
@@ -0,0 +1,161 @@
+/*
+   BAREOS® - Backup Archiving REcovery Open Sourced
+
+   Copyright (C) 2018-2020 Bareos GmbH & Co. KG
```

Comment:
```suggestion
   Copyright (C) 2020-2020 Bareos GmbH & Co. KG
```
it's a new file, isn't it?